### DISCUSSION TITLE
Is there a way to prevent hardware reset by repeated power ON/OFF

### DISCUSSION THREAD

**Discussion by schwarzelise** [arendst/Tasmota#21744] [Q&A] [answered]
Hi there,
i had a problem with my ATS in my solar system. It switched many times form ON to OFF and back and killed my configuration of about 50 Tasmota devices. All needed attached newly to wifa and all params to be set(nightmare)

Question is there a option in the settings to prevent that. HowTo best save and restore the data?

Thx in Adv

**Comment by sfromis** [comment, marked as answer]
Yes there is, `SetOption65`

**Reply by schwarzelise** [reply]
Sorry i searched for wrong term. 
Thx a lot
